simplified-main/src/main/java/org/nypl/simplified/main/MainActivity.kt

```diff
@@ -91,6 +91,10 @@ class MainActivity : AppCompatActivity(R.layout.main_host) {
     }
 
     pendingLink.addOnSuccessListener { linkData ->
+      if (linkData == null) {
```

Comment:
I'd suggest to do `val deepLink = linkData?.link` because that way we wouldn't more any more if conditions